### ISSUE TITLE
Can we make the trees serializable?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [npgall/concurrent-trees#4]
```
The current implementation is not serializable. If we load a huge amount of 
data each time when starting, this may limit the usage. However, if it is 
serializable, we can load it once and serialize the entire tree onto the disk. 
During the start-up time, we only have to de-serialize it to load the whole 
tree quickly.
```

Original issue reported on code.google.com by `j_yuz...@yahoo.com` on 25 Mar 2013 at 2:29

**Comment by w4nderlust** [comment]
Upvote for this feature. It's really needed.

**Comment by jcastaldoFoodEssentials** [comment]
Hey I saw you committed code to make this a reality. Any idea on when you're planning to release 2.5.0?

**Comment by npgall** [comment]
I'm planning to release 2.5.0 in the next 1-2 weeks :)

**Comment by jcastaldoFoodEssentials** [comment]
My team and I are very excited!

**Comment by visheshjoshi** [comment]
@npgall Thanks for adding the serialization. Really needed for my project. I hope you release it soon.

**Comment by npgall** [comment]
I have released 2.5.0. It should sync to Maven Central in ~2 hours.

**Comment by npgall** [comment]
Version 2.5.0 is now in Maven Central. Closing this issue.
